README.md

```diff
@@ -111,6 +111,9 @@ bumpversion $$VERSION_PART_TO_BUMP$$
 git push && git push --tags
 make release
 ```
+or
+#To run on a container
```

Comment:
@divyanks I think this isn't the best place to address this.  I think we should rather add a `Installing via Docker` section here:

https://github.com/ethereum/py-evm/blob/30b35f79cf299cfbab4c13a70d55b808fb6b26b2/docs/guides/trinity/quickstart.rst